ui/components/src/context/ChartsThemeProvider.tsx

```diff
@@ -12,7 +12,7 @@
 // limitations under the License.
 
 import React, { createContext, useContext } from 'react';
-import { registerTheme } from 'echarts';
+import { registerTheme } from 'echarts/core';
```

Comment:
This was causing all of `echarts` to get pulled in when you used the `ChartThemeProvider` 😂